andela/ah-django-unchained-frontend
# #162948806 get one article

#### What does this PR do?
Enable users to fetch single article via a slug

#### Description of Task to be completed?
* [x]   Create actions for fetching a single article
* [x]   Create reducer for fetching a single article
* [x]   Consume the `get  single article` API endpoint
* [x]   Created test for the front end

#### How should this be manually tested?
1. Clone this repository on your local machine using
   `git clone https://github.com/andela/ah-django-unchained-frontend.git`
2. Change your working directory to the project's directory using
   `cd ah-django-unchained-frontend`
3. Change to the specific branch `ft-get-one-article-1629488 ` using
   `git checkout ft-get-one-article-16294880 `
4. Create a `.env` file and inside it add this line `REACT_APP_BASE_URL=app-base-url` where `app-base-url ` is the root url of the backend API
5. Run `npm install` to install all the modules listed in the `package.json` file.
6. Run `npm start` command to start the application.
7. The application should start successfully. On the browser, navigate to the URL `http://localhost:3000/article/artilce-slug`, .

#### Link to PT story
[#162948806](https://www.pivotaltracker.com/story/show/162948806)

#### Screenshots
<img width="1280" alt="screenshot 2019-02-27 at 22 40 47" src="https://user-images.githubusercontent.com/30651751/53517883-c3964000-3ae0-11e9-9775-856aa568804b.png">

<img width="1280" alt="screenshot 2019-02-27 at 22 41 00" src="https://user-images.githubusercontent.com/30651751/53517904-cf820200-3ae0-11e9-90dd-610363ad25a6.png">
